_sass/_layout.scss

```diff
@@ -35,14 +39,68 @@
     padding: 0 10px;
     line-height: 63px;
   }
+
+  .hamburger-site-nav {
+    @include border-radius(4px);
+    @include media_min(1031px) { display: none; }
+    position: fixed; z-index: 999;
+    top: 12px; right: 26px;
+    background: $green;
+    border: 1px solid rgba(#333,.2);
+
+    .nav-trigger {
+      @include border-radius(4px);
+      padding: 8px;
+      display: block;
+      text-align: right;
+      line-height: 1;
+
+      .menu-icon { display: block; }
+    }
+
+    .links {
+      list-style: none;
+      margin: 0;
+      padding: 0 15px 25px;
```

Comment:
Lets reduce the bottom padding significantly, because having them so spaced out is going to make it harder to navigate on mobile (they will likely exceed the viewport with the current overall height). Maybe just 3-5px padding on the bottom?